WPMedia/fusion-news-theme-blocks
# TMEDIA 494: Results and search results list refactor of reusable button

## Description
The Results List block and Search Results List button could be updated to use the new button components for theme consistency.

Included the search results list button as well because it is using shared styles

## Jira Ticket
- [TMEDIA-494](https://arcpublishing.atlassian.net/browse/TMEDIA-494)

## Acceptance Criteria

1.  Button is utilizing the Primary Label button

## Test Steps

1. Checkout this branch `git checkout TMEDIA-494-results-list-theme-button`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/results-list-block,@wpmedia/search-results-list-block`
3. Go to search results page http://localhost/search/video/?_website=the-sun
4. See button work as expected on click and on hover
5. Go to results page http://localhost/pf/results-list/?_website=the-sun
6. See button work as expected on click and on hover

## Effect Of Changes
### Before
- search results button doesn't match themes (has border, wrong hover) https://corecomponents.arcpublishing.com/pf/search/video/?_website=the-sun

hover
<img width="712" alt="Screen Shot 2022-01-03 at 10 40 50" src="https://user-images.githubusercontent.com/5950956/147956248-209440fc-8ff4-4ee7-8a3a-bda477718f7e.png">
<img width="869" alt="Screen Shot 2022-01-03 at 10 40 41" src="https://user-images.githubusercontent.com/5950956/147956271-76aecd99-3b0d-4281-81a5-aeb7f2b18410.png">

results list button https://corecomponents.arcpublishing.com/pf/results-list/?_website=the-sun
<img width="934" alt="Screen Shot 2022-01-03 at 10 44 43" src="https://user-images.githubusercontent.com/5950956/147956555-79406294-633e-4cb1-a131-f2d3f6ef9df7.png">
does not darken on hover


### After

- search results button matches designs (no border, no weird hover color) http://localhost/search/video/?_website=the-sun
<img width="849" alt="Screen Shot 2022-01-03 at 10 39 32" src="https://user-images.githubusercontent.com/5950956/147956083-c494934a-cc68-4d45-a272-91aec656ecbf.png">
hover 
<img width="190" alt="Screen Shot 2022-01-03 at 10 42 23" src="https://user-images.githubusercontent.com/5950956/147956315-162f05b5-c364-4de1-be5c-fa13d48a7656.png">

local results list http://localhost/pf/results-list/?_website=the-sun
- does darken on hover
<img width="169" alt="Screen Shot 2022-01-03 at 10 46 33" src="https://user-images.githubusercontent.com/5950956/147956732-ea5378af-1971-42c9-8ec5-41526e7875c9.png">
- uses reusable button, no border 
<img width="902" alt="Screen Shot 2022-01-03 at 10 46 57" src="https://user-images.githubusercontent.com/5950956/147956778-0fe53ef6-632c-4f60-a445-f204a694874e.png">

## Dependencies or Side Effects
- Deleted unused css 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
